redis-test/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "redis-test"
+version = "0.21.5"
+edition = "2021"
+description = "Testing helpers for the `redis` crate"
+homepage = "https://github.com/redis-rs/redis-rs"
+repository = "https://github.com/redis-rs/redis-rs"
+documentation = "https://docs.rs/redis-test"
+license = "BSD-3-Clause"
+
+[dependencies]
+redis = { path = ".." }
```

Comment:
If we're going to publish this it should also have a specific version.

Reply:
I added a `version`, but kept the `path` so it continues to work with in-repo builds. Tokio does something similar.